File: solutions/go/sum-of-multiples/1/sum_of_multiples.go
package summultiples

func SumMultiples(limit int, divisors ...int) (sum int) {
	for number := 1; number < limit; number++ {
		for _, divisor := range divisors {
			if divisor != 0 && number%divisor == 0 {
				sum += number
				break
			}
		}
	}
	return
}

File: solutions/python/sum-of-multiples/1/sum_of_multiples.py
def sum_of_multiples(limit, multiples):
    if 0 in multiples:
        multiples.remove(0)
    sum_of_multiples_to_return = 0
    for number in range(limit):
        for divisor in multiples:
            if not (number % divisor):
                sum_of_multiples_to_return += number
                break
    return sum_of_multiples_to_return
